docs/loadflow/parameters.md

```diff
@@ -112,6 +112,21 @@ When slack distribution is enabled (`distributedSlack` set to `true` in LoadFlow
 is considered to be distributed.  
 The default value is `1 MW` and it must be greater or equal to `0 MW`.
 
+**slackDistributionGeneratorsVoltageControlOnly**  
+When set to `true`, only generators on voltage control will participate in slack distribution.
+
+Notes:
+* Effective only when in LoadFlowParameters:
+  1. `distributedSlack` is set to `true`, and,
+  2. `balanceType` is set to a mode distributing on generators
+* If a generator voltage control is enabled in input data but internally disabled due to a consistency check,
+(see e.g. `reactiveRangeCheckMode` and `disableVoltageControlOfGeneratorsOutsideActivePowerLimits`), 
+then no slack distribution will occur for that generator.

```

Comment:
not sure this is right, changing PR to WIP

Reply:
this is not right, or say, not what I need. I need to consider only the input setting of the generator.